frontend/src/components/pages/CreateGame/views/ChooseGameMode.js

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+import Util from "../../../../util/Util";
+import GameUtil from "../../../../util/GameUtil";
+import Loader from "../../../misc/Loader";
+import GameMode from "../components/GameMode";
+import Title from "../../../misc/Title";
+
+
+export default class ChooseGameMode extends React.Component {
+
+    static TITLE = 'Choisissez un mode de jeu';
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            gameConfiguration: Util.getObjectFromSessionStorage(GameUtil.GAME_CONFIGURATION.key),
+            gamesModes: false,
+            isLoading: true,
+        };
+    }
+
+    componentDidMount() {
+        (async () => {
+            try {
+                const gameModes = await this.getGameModes();
+
+                this.setState({
+                    gameModes,
+                    isLoading: false
+                });
+            } catch (error) {
+                console.error(error);
+            }
+        })();
+    }
+
+    getGameModes = async () => {
+        try {
+            const userPlan = Util.getJwtPayloadContent(Util.accessToken).user.plan;
+
+            const response = await Util.sendJsonToAPI('game/modes', {plan: userPlan});
+
+            if (!response.ok) throw `${response.status} : ${response.statusText}`;
+
+            const responseData = await response.json();
+
+            return responseData.gameModes;
+        } catch (error) {
+            throw error;
+        }
```

Comment:
Bad usage of try/catch : this catch is useless

Reply:
You didn't know this hack ? its called "twice catching".

No more jokes, I fix that asap